src/components/history/RequestsListItem.tsx

```diff
@@ -0,0 +1,22 @@
+import { Button, Link, ListItem } from '@mui/material';
```

Comment:
```suggestion
import Link from 'next/link';
import { Button, ListItem } from '@mui/material';
```
using the Link component from mui reloads the page on navigation